src/client/Client.cc

```diff
@@ -2406,8 +2419,8 @@ void Client::handle_client_session(const MConstRef<MClientSession>& m)
       if (!session->seq && m->get_seq())
         session->seq = m->get_seq();
 
-      session->mds_features = std::move(m->supported_features);
-      session->mds_metric_flags = std::move(m->metric_spec.metric_flags);
+      reinit_mds_features(session.get(), m);
+      cap_auths = std::move(m->cap_auths);
```

Comment:
Venky, this is incorrect. Please remove this line.

Reply:
oh, auto conflict resolution :/